pandas/core/frame.py

```diff
@@ -2102,22 +2102,24 @@ def to_feather(self, path, **kwargs) -> None:
 
         to_feather(self, path, **kwargs)
 
-    @Appender(
-        """
-        Examples
-        --------
-        >>> df = pd.DataFrame(
-        ...     data={"animal_1": ["elk", "pig"], "animal_2": ["dog", "quetzal"]}
-        ... )
-        >>> print(df.to_markdown())
-        |    | animal_1   | animal_2   |
-        |---:|:-----------|:-----------|
-        |  0 | elk        | dog        |
-        |  1 | pig        | quetzal    |
-        """
+    @doc(
+        Series.to_markdown,
+        klass=_shared_doc_kwargs["klass"],
```

Comment:
Shouldn’t klass here be DataFrame?

Reply:
`_shared_doc_kwargs["klass"]=="DataFrame"` Using the dictionary is more consistent with other places in this file where the decorator is used.

Reply:
@WillAyd do you accept my reasoning?